packages/cli/snippets/middleware.js

```diff
@@ -10,6 +10,26 @@ const mustBe200 = (response, z, bundle) => {
   return response;
 };
 
+const handleErrors = (response, z) => {
```

Comment:
Updated the middleware snippet to show:

1. Using `afterResponse` makes you responsible for error response handling.
2. How to throw for statuses that `throwForStatus` won't (correctly).
3. How to prevent `throwForStatus` from throwing.
4. That you should always call `throwForStatus`.